iree/tools/CMakeLists.txt

```diff
@@ -14,15 +14,42 @@
 
 add_subdirectory(test)
 
-# TODO: iree_select_target_platform macros.
-# TODO: skip targets disabled by options.
-iree_select_compiler_opts(IREE_HAL_DRIVER_MODULES
-  ALL
-    "iree::hal::dylib::dylib_driver_module"
-    "iree::hal::llvmjit::llvmjit_driver_module"
-    "iree::hal::vmla::vmla_driver_module"
-    "iree::hal::vulkan::vulkan_driver_module"
-)
+# Enable HAL driver modules based on options.
```

Comment:
Probably good to add `bazel_to_cmake: DO NOT EDIT`. This is a new reason why bazel_to_cmake can't handle this file.

Reply:
Add it to the list :P I'll add some notes. Since this file pulls in all sorts of components, I don't see us getting it running fully through the conversion tool anytime soon.